packages/skills/Extras.js

```diff
@@ -860,10 +860,10 @@ extrasList = {
 			targ.mp -= MPtaken;
 			if (targ.mp < 0) targ.mp = 0;
 
-			return `__${char.name}__ took **${MPtaken} MP** from __${targ.name}__!`;
+			return `__${char.name}__ took **${MPtaken}${char.mpMeter[1]}** from __${targ.name}__!`;
```

Comment:
Should this be using the user's MP meter (like it does now), or the foes'?

Reply:
What do you mean?

Reply:
In Felix's case, this would say "Felix took 40DP from Grassimp", but Grassimps do not use DP. Should this be "Felix took 40MP from Grassimp?"

Reply:
Hm...I see...
What would make more sense?

Reply:
i dunno thats why im asking

Reply:
That, or you push, and i rework the message

Reply:
probably best to let you rework it as you prefer, if you want that

Reply:
Okie, ill do that then